README.md

```diff
@@ -32,7 +32,7 @@ yarn install
 
 - 🤝 [Greeter](./packages/greeter/) (parent to child messages)
 - 📤 [Outbox](./packages/outbox-execute/) (child to parent messages)
-- ⏰ [L1 Confirmation Checker](./packages/l1-confirmation-checker/)
+- ⏰ [Parent chain confirmation checker](./packages/l1-confirmation-checker/)
```

Comment:
Not sure if we need to rename the package name there too. (from `l1-confirmation-checker` to `parent-confirmation-checker`)